### PR TITLE
feat(workerpool): bring up

### PR DESCRIPTION
**Pull request by SukkaW**
The PR is a part of hexojs/hexo#4355.

Bring up a utility to manage workers and job queue.

### CODE REVIEW

**Comment by SukkaW** [comment]
Minimum required Node.js version has bumped to 12.13, which is the first LTS version of Node.js 12: https://nodejs.org/en/blog/release/v12.13.0/

File: scripts/dummy_worker.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// eslint-disable-next-line node/no-unsupported-features/node-builtins
```
Comment (curbengh) [inline scripts/dummy_worker.js:3]: not required after https://github.com/hexojs/hexo-util/pull/212/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R57
comment in other files can be removed too.

**Comment by SukkaW** [comment]
@curbengh We might not bring up Worker Threads during Hexo 5.0.0 development. So we can leave this PR here until we officially drop Node.js 12.

**Comment by stevenjoezhang** [comment]
Any updates on this thread?

**Pull request closed without merging**